joplin/js/CreateContentModal/index.js

```diff
@@ -50,16 +50,22 @@ class CreateContentModal extends Component {
 
   incrementActiveStep = () => {
     // If we're on the choose type step
-    if(this.state.activeStep === stepsEnum.CHOOSE_TYPE) {
+    if (this.state.activeStep === stepsEnum.CHOOSE_TYPE) {
       // Department pages, topic pages, and service pages go straight to title
-      if(this.state.type === 'department' || this.state.type === 'service' || this.state.type === 'topic') {
+      if (
```

Comment:
I definitely want to do another create content modal dive soon, it could use some love